### DISCUSSION TITLE
Use-case idea

### DISCUSSION THREAD

**Discussion by sandstrom** [netbirdio/netbird#892] [Ideas & Feature Requests]
Just wanted to share a use-case that I think you can promote on the website, once the iOS and Android clients has landed.

Our app developers need to test stuff on mobile devices, against their local machine. Right now, they have to punch holes in their macOS firewall, open ports and change interface bindings (by default we prefer to only bind to loopback).

With netbird, ideally, they could setup a tunnel between their mobile device and developer computer, and have the mobile app use that tunnel to access the backend service running on the laptop.

Maybe it's just us, but I think this might be useful to others too!

**Comment by mlsmaycon** [comment]
Hello @sandstrom thanks for your use-case, I think it is very valid and we see ourselves doing the same thing with the android client development and with some of our recent features like ACL. 

Currently we released the first public version of our Android application: https://play.google.com/store/apps/details?id=io.netbird.client

It doesn't support DNS yet, so I am not sure how much you can validate the idea, but in case you have an internal DNS which the mobile device has access too, you can probably create some records to help. 

Let us know if you think this can support your initial tests.

**Reply by sandstrom** [reply]
Sounds great!

I think it's too early, but I'll check back when you've released the iOS app.
